docs/methods/preprocessing/label_encoder.md

```diff
@@ -0,0 +1,19 @@
+# Label Encoder
+
+Encode target labels with value between 0 and n_classes-1.
+
+This transformer should be used to encode target values, i.e. y, and not the input X.
+
```

Comment:
Improve the description since the term `n_classes` is very vague

Reply:
tried to improve it